main.go

```diff
@@ -166,7 +134,7 @@ func getConfig() (*config, error) {
 	if ok {
 		repositoryWatchListSync = strings.Split(watchRepositoriesGitLabSync, ",")
 	} else {
-		repositoryWatchListSync = defaultWatchRepositoriesSync
+		repositoryWatchListSync = nil
 	}
```

Comment:
I think we can completely drop support for `WATCH_REPOS_SYNC` env variable. I would completely remove this block and [continues in next comment]